packages/arb-token-bridge-ui/next.config.js

```diff
@@ -7,5 +7,28 @@
 module.exports = {
   distDir: 'build',
   productionBrowserSourceMaps: true,
-  reactStrictMode: true
+  reactStrictMode: true,
+  async headers() {
+    return [
+      {
+        source: '/api/status',
+        headers: [
+          { key: 'Access-Control-Allow-Credentials', value: 'false' },
+          {
+            key: 'Access-Control-Allow-Origin',
+            value: 'https://portal.arbitrum.io/'
+          },
+          {
+            key: 'Access-Control-Allow-Methods',
+            value: 'GET'
+          },
+          {
+            key: 'Access-Control-Allow-Headers',
+            value:
+              'Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date'
+          }
```

Comment:
We can just get rid of this entirely as the necessary headers are safelisted (https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header)